src/pages/donate.js

```diff
@@ -66,7 +66,9 @@ const Donate = ({ componentName }) => {
 								We love our lattes. We also love knowing that our youth are
 								prepared for the future. $5 matters. You matter. Let’s do this.
 							</p>
+							<a href = "https://paypal.me/treobytes" target = "_blank">
```

Comment:
did you test your code locally? Wrapping the button with an anchor tag alters the width of the button. 
You'll need to update the styles for the donate page and style this anchor tag. You can give the anchor tag(s) the className of "btn-wrapper"

Reply:
I did test the code locally and saw the width of the button being affected. I am still working to get used to the code and am not able to correct the width of the button yet. So, I was intending to use the smaller width button initially and fix it in a future update. 

I added the className 'btn-wrapper' and am looking to update the styles of the donate page.

Reply:
I spoke to Ava and she agreed to temporarily go live with the site having the smaller buttons and they can be fixed later.